qa/tasks/rgw.py

```diff
@@ -198,6 +198,59 @@ def start_rgw(ctx, config, clients):
                 exit_on_first_error=False
                 )
 
+        cloudtier_config = client_config.get('cloudtier')
+        if cloudtier_config is not None:
```

Comment:
Agreed!

Reply:
having two cloudtier storage classes as part of same task is resulting in spurious failures often. Hence I have disabled one configuration for now but still retained the code in case we find any consistent way of testing lifecycle cases in future.